version_bump.py

```diff
@@ -0,0 +1,67 @@
+### INSTRUCTIONS ###
+
+# To bump the current version of u8timeseries, please execute the version bumping script from the root:
+#
+# >>> python version_bump.py
+# 
+# The user will then be prompted to enter the desired bump settings.
+# Alternatively, the settings can also be passed as arguments when calling the script:
+#
+# >>> python version_bump.py -b BUMP -r RELEASE -p PATH
+#
+# BUMP should be an integer between 0 and 3 indicating the type of version bump.
+# (0 for no version increment, 1 for Major, 2 for Minor, 3 for Patch)
+# RELEASE is expected to be 'y' or 'n', indicating whether or not the SNAPSHOT suffix should be added.
+# If either of the arguments is missing, or if an invalid value is given, the user will be prompted
+# to enter them again.
+# PATH should be the path to the version file that needs to be bumped.
+#
+# The versioning of u8timeseries follows the semantic versioning specification (https://semver.org).
+
+import argparse
+
+VERSION_FILE = "u8timeseries/VERSION"
+
+# set up arguments
+parser = argparse.ArgumentParser(prog="version bump")
+parser.add_argument('-b', '--bump', help="BUMP should be an integer between 0 and 3" \
+                                         " indicating the type of version bump.")
+parser.add_argument('-r', '--release', help="RELEASE is expected to be 'y' or 'n', indicating" \
+                                            " whether or not the SNAPSHOT suffix should be added.")
```

Comment:
I'm nitpicking again :) but here it could be read as if `y` = adding `SNAPSHOT`.
I'd rather write: `RELEASE is expected to be 'y' or 'n'. If set to 'n' the SNAPSHOT suffix will be added.`